[2022-07-28 20:58:57.416] [remoteagent] [info] Extension host agent started.
[2022-07-28 20:58:57.908] [remoteagent] [info] [127.0.0.1][25d3143e][ManagementConnection] New connection established.
[2022-07-28 20:58:57.993] [remoteagent] [info] [127.0.0.1][29edd8a8][ExtensionHostConnection] New connection established.
[2022-07-28 20:58:57.997] [remoteagent] [info] [127.0.0.1][29edd8a8][ExtensionHostConnection] <13045> Launched Extension Host Process.
[2022-07-28 20:59:41.811] [remoteagent] [info] [127.0.0.1][29edd8a8][ExtensionHostConnection] <13045> Extension Host Process exited with code: 0, signal: null.
[2022-07-28 20:59:41.811] [remoteagent] [info] Last EH closed, waiting before shutting down
[2022-07-28 20:59:45.559] [remoteagent] [info] Got delay-shutdown request while in shutdown timeout, delaying
[2022-07-28 20:59:45.559] [remoteagent] [info] Cancelling previous shutdown timeout
[2022-07-28 20:59:46.058] [remoteagent] [info] [127.0.0.1][55a2983e][ManagementConnection] New connection established.
[2022-07-28 20:59:46.228] [remoteagent] [info] [127.0.0.1][9bf71daf][ExtensionHostConnection] New connection established.
[2022-07-28 20:59:46.234] [remoteagent] [info] [127.0.0.1][9bf71daf][ExtensionHostConnection] <13188> Launched Extension Host Process.
[2022-07-28 20:59:46.746] [remoteagent] [info] [127.0.0.1][25d3143e][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
[2022-07-28 20:59:46.760] [remoteagent] [error] Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/server/node/server.main.js:145:7933)
    at process.emit (node:events:390:28)
[2022-07-28 21:00:43.710] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-28 21:01:02.193] [remoteagent] [info] [127.0.0.1][55a2983e][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
[2022-07-28 21:01:05.462] [remoteagent] [error] Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/server/node/server.main.js:145:7933)
    at process.emit (node:events:390:28)
[2022-07-28 21:01:09.642] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-28 21:01:24.464] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-28 21:01:32.617] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-28 21:01:42.493] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-28 21:01:50.067] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-28 21:02:00.959] [remoteagent] [info] [127.0.0.1][9bf71daf][ExtensionHostConnection] <13188> Extension Host Process exited with code: 0, signal: null.
[2022-07-28 21:02:00.962] [remoteagent] [info] Cancelling previous shutdown timeout
[2022-07-28 21:02:00.963] [remoteagent] [info] Last EH closed, waiting before shutting down
[2022-07-28 21:02:01.520] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateIcon (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25409)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-28 21:02:01.521] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-28 21:02:01.521] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-28 21:05:57.315] [remoteagent] [info] Got delay-shutdown request while in shutdown timeout, delaying
[2022-07-28 21:05:57.315] [remoteagent] [info] Cancelling previous shutdown timeout
[2022-07-28 21:05:57.754] [remoteagent] [info] [127.0.0.1][25d3143e][ManagementConnection] Another client has connected, will shorten the wait for reconnection 5m before disposing...
[2022-07-28 21:05:57.754] [remoteagent] [info] [127.0.0.1][55a2983e][ManagementConnection] Another client has connected, will shorten the wait for reconnection 5m before disposing...
[2022-07-28 21:05:57.754] [remoteagent] [info] [127.0.0.1][785f17c0][ManagementConnection] New connection established.
[2022-07-28 21:05:57.932] [remoteagent] [info] [127.0.0.1][66830359][ExtensionHostConnection] New connection established.
[2022-07-28 21:05:57.937] [remoteagent] [info] [127.0.0.1][66830359][ExtensionHostConnection] <13489> Launched Extension Host Process.
[2022-07-28 21:05:59.978] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-28 21:06:17.728] [remoteagent] [info] [127.0.0.1][66830359][ExtensionHostConnection] <13489> Extension Host Process exited with code: 0, signal: null.
[2022-07-28 21:06:17.728] [remoteagent] [info] Cancelling previous shutdown timeout
[2022-07-28 21:06:17.728] [remoteagent] [info] Last EH closed, waiting before shutting down
[2022-07-28 21:06:21.311] [remoteagent] [info] Got delay-shutdown request while in shutdown timeout, delaying
[2022-07-28 21:06:21.311] [remoteagent] [info] Cancelling previous shutdown timeout
[2022-07-28 21:06:21.704] [remoteagent] [info] [127.0.0.1][89d259b3][ManagementConnection] New connection established.
[2022-07-28 21:06:21.916] [remoteagent] [info] [127.0.0.1][95635681][ExtensionHostConnection] New connection established.
[2022-07-28 21:06:21.948] [remoteagent] [info] [127.0.0.1][95635681][ExtensionHostConnection] <13629> Launched Extension Host Process.
[2022-07-28 21:06:22.331] [remoteagent] [info] [127.0.0.1][785f17c0][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
[2022-07-28 21:06:22.338] [remoteagent] [error] Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/server/node/server.main.js:145:7933)
    at process.emit (node:events:390:28)
[2022-07-28 21:06:23.871] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-28 21:06:34.281] [remoteagent] [info] [127.0.0.1][95635681][ExtensionHostConnection] <13629> Extension Host Process exited with code: 0, signal: null.
[2022-07-28 21:06:34.281] [remoteagent] [info] Cancelling previous shutdown timeout
[2022-07-28 21:06:34.281] [remoteagent] [info] Last EH closed, waiting before shutting down
[2022-07-28 21:06:45.625] [remoteagent] [info] Got delay-shutdown request while in shutdown timeout, delaying
[2022-07-28 21:06:45.626] [remoteagent] [info] Cancelling previous shutdown timeout
[2022-07-28 21:06:45.963] [remoteagent] [info] [127.0.0.1][785f17c0][ManagementConnection] Another client has connected, will shorten the wait for reconnection 5m before disposing...
[2022-07-28 21:06:45.964] [remoteagent] [info] [127.0.0.1][6e20e816][ManagementConnection] New connection established.
[2022-07-28 21:06:46.093] [remoteagent] [info] [127.0.0.1][92b80f4b][ExtensionHostConnection] New connection established.
[2022-07-28 21:06:46.097] [remoteagent] [info] [127.0.0.1][92b80f4b][ExtensionHostConnection] <13748> Launched Extension Host Process.
[2022-07-28 21:06:48.804] [remoteagent] [info] [127.0.0.1][89d259b3][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
[2022-07-28 21:06:48.806] [remoteagent] [error] Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/server/node/server.main.js:145:7933)
    at process.emit (node:events:390:28)
[2022-07-28 21:07:08.682] [remoteagent] [info] [127.0.0.1][92b80f4b][ExtensionHostConnection] <13748> Extension Host Process exited with code: 0, signal: null.
[2022-07-28 21:07:08.682] [remoteagent] [info] Cancelling previous shutdown timeout
[2022-07-28 21:07:08.683] [remoteagent] [info] Last EH closed, waiting before shutting down
[2022-07-28 21:07:12.400] [remoteagent] [info] Got delay-shutdown request while in shutdown timeout, delaying
[2022-07-28 21:07:12.400] [remoteagent] [info] Cancelling previous shutdown timeout
[2022-07-28 21:07:12.709] [remoteagent] [info] [127.0.0.1][89d259b3][ManagementConnection] Another client has connected, will shorten the wait for reconnection 5m before disposing...
[2022-07-28 21:07:12.709] [remoteagent] [info] [127.0.0.1][9880efbe][ManagementConnection] New connection established.
[2022-07-28 21:07:13.046] [remoteagent] [info] [127.0.0.1][992f82f1][ExtensionHostConnection] New connection established.
[2022-07-28 21:07:13.052] [remoteagent] [info] [127.0.0.1][992f82f1][ExtensionHostConnection] <13889> Launched Extension Host Process.
[2022-07-28 21:07:13.363] [remoteagent] [info] [127.0.0.1][6e20e816][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
[2022-07-28 21:07:13.365] [remoteagent] [error] Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/server/node/server.main.js:145:7933)
    at process.emit (node:events:390:28)
[2022-07-28 21:07:29.503] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-28 21:10:22.025] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-28 21:12:06.527] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-28 21:13:17.414] [remoteagent] [info] New EH opened, aborting shutdown
[2022-07-28 21:13:17.495] [remoteagent] [info] [127.0.0.1][9880efbe][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
[2022-07-28 21:13:17.496] [remoteagent] [error] Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/server/node/server.main.js:145:7933)
    at process.emit (node:events:390:28)
[2022-07-28 21:13:17.781] [remoteagent] [info] [127.0.0.1][992f82f1][ExtensionHostConnection] <13889> Extension Host Process exited with code: 0, signal: null.
[2022-07-28 21:13:17.781] [remoteagent] [info] Last EH closed, waiting before shutting down
[2022-07-28 21:13:18.472] [remoteagent] [info] Got delay-shutdown request while in shutdown timeout, delaying
[2022-07-28 21:13:18.472] [remoteagent] [info] Cancelling previous shutdown timeout
[2022-07-28 21:13:18.487] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateIcon (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25409)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-28 21:13:18.488] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-28 21:13:18.495] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-28 21:13:18.812] [remoteagent] [info] [127.0.0.1][6e20e816][ManagementConnection] Another client has connected, will shorten the wait for reconnection 5m before disposing...
[2022-07-28 21:13:18.812] [remoteagent] [info] [127.0.0.1][9880efbe][ManagementConnection] Another client has connected, will shorten the wait for reconnection 5m before disposing...
[2022-07-28 21:13:18.813] [remoteagent] [info] [127.0.0.1][00e406e5][ManagementConnection] New connection established.
[2022-07-28 21:13:18.893] [remoteagent] [info] [127.0.0.1][ef8d6eeb][ExtensionHostConnection] New connection established.
[2022-07-28 21:13:18.898] [remoteagent] [info] [127.0.0.1][ef8d6eeb][ExtensionHostConnection] <14041> Launched Extension Host Process.
[2022-07-28 21:13:21.116] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-28 21:14:26.075] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-28 21:15:19.144] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-28 21:16:48.335] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-28 21:17:06.375] [remoteagent] [info] [127.0.0.1][00e406e5][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
[2022-07-28 21:17:12.963] [remoteagent] [error] Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/server/node/server.main.js:145:7933)
    at process.emit (node:events:390:28)
[2022-07-28 21:17:17.507] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-28 21:17:35.523] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-28 21:17:48.882] [remoteagent] [info] [127.0.0.1][ef8d6eeb][ExtensionHostConnection] <14041> Extension Host Process exited with code: 0, signal: null.
[2022-07-28 21:17:48.884] [remoteagent] [info] Cancelling previous shutdown timeout
[2022-07-28 21:17:48.884] [remoteagent] [info] Last EH closed, waiting before shutting down
[2022-07-28 21:17:49.709] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateIcon (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25409)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-28 21:17:49.772] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-28 21:17:49.779] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-28 21:18:03.333] [remoteagent] [info] Got delay-shutdown request while in shutdown timeout, delaying
[2022-07-28 21:18:03.333] [remoteagent] [info] Cancelling previous shutdown timeout
[2022-07-28 21:18:03.679] [remoteagent] [info] [127.0.0.1][00e406e5][ManagementConnection] Another client has connected, will shorten the wait for reconnection 5m before disposing...
[2022-07-28 21:18:03.679] [remoteagent] [info] [127.0.0.1][6525a01c][ManagementConnection] New connection established.
[2022-07-28 21:18:03.811] [remoteagent] [info] [127.0.0.1][2550958d][ExtensionHostConnection] New connection established.
[2022-07-28 21:18:03.816] [remoteagent] [info] [127.0.0.1][2550958d][ExtensionHostConnection] <14332> Launched Extension Host Process.
[2022-07-28 21:18:06.302] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-28 21:18:22.864] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-28 21:18:37.879] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-28 21:18:48.955] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-28 21:19:02.899] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-28 21:19:35.309] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-28 21:20:08.499] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-28 21:20:35.172] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-28 21:20:52.181] [remoteagent] [info] [127.0.0.1][6525a01c][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
[2022-07-28 21:21:00.177] [remoteagent] [error] No ptyHost heartbeat after 12 seconds
[2022-07-28 21:21:10.300] [remoteagent] [error] Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/server/node/server.main.js:145:7933)
    at process.emit (node:events:390:28)
[2022-07-28 21:21:13.711] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-28 21:21:25.968] [remoteagent] [error] No ptyHost heartbeat after 12 seconds
[2022-07-28 21:21:43.808] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-28 21:21:52.446] [remoteagent] [error] No ptyHost heartbeat after 12 seconds
[2022-07-28 21:22:10.846] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-28 21:22:21.952] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-28 21:22:29.018] [remoteagent] [error] No ptyHost heartbeat after 12 seconds
[2022-07-28 21:22:56.676] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateIcon (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25409)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-28 21:22:56.685] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-28 21:22:56.685] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-28 21:23:05.988] [remoteagent] [info] New EH opened, aborting shutdown
[2022-07-28 21:23:07.150] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-28 21:23:19.123] [remoteagent] [info] [127.0.0.1][6525a01c][ManagementConnection] Another client has connected, will shorten the wait for reconnection 5m before disposing...
[2022-07-28 21:23:19.142] [remoteagent] [info] [127.0.0.1][99f351a8][ManagementConnection] New connection established.
[2022-07-28 21:23:19.142] [remoteagent] [info] [127.0.0.1][00fda346][ExtensionHostConnection] New connection established.
[2022-07-28 21:23:19.317] [remoteagent] [info] [127.0.0.1][00fda346][ExtensionHostConnection] <14683> Launched Extension Host Process.
[2022-07-28 21:25:03.130] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-28 21:25:41.233] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-28 21:26:10.376] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-28 21:26:44.213] [remoteagent] [error] No ptyHost heartbeat after 12 seconds
[2022-07-28 21:29:01.303] [remoteagent] [info] [127.0.0.1][99f351a8][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
[2022-07-28 21:29:03.319] [remoteagent] [error] Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/server/node/server.main.js:145:7933)
    at process.emit (node:events:390:28)
[2022-07-28 21:29:06.722] [remoteagent] [info] [127.0.0.1][25d3143e][ManagementConnection] The reconnection short grace time of 5m has expired, so the connection will be disposed.
[2022-07-28 21:29:06.766] [remoteagent] [info] [127.0.0.1][55a2983e][ManagementConnection] The reconnection short grace time of 5m has expired, so the connection will be disposed.
[2022-07-28 21:29:22.376] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateIcon (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25409)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-28 21:29:26.789] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-28 21:29:31.677] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-28 21:30:51.899] [remoteagent] [info] [127.0.0.1][785f17c0][ManagementConnection] The reconnection short grace time of 5m has expired, so the connection will be disposed.
[2022-07-28 21:31:39.361] [remoteagent] [info] [127.0.0.1][89d259b3][ManagementConnection] The reconnection short grace time of 5m has expired, so the connection will be disposed.
[2022-07-28 21:32:14.562] [remoteagent] [info] [127.0.0.1][6e20e816][ManagementConnection] The reconnection short grace time of 5m has expired, so the connection will be disposed.
[2022-07-28 21:32:30.511] [remoteagent] [info] [127.0.0.1][9880efbe][ManagementConnection] The reconnection short grace time of 5m has expired, so the connection will be disposed.
[2022-07-28 21:34:19.274] [remoteagent] [info] [127.0.0.1][00e406e5][ManagementConnection] The reconnection short grace time of 5m has expired, so the connection will be disposed.
[2022-07-28 21:38:58.827] [remoteagent] [info] [127.0.0.1][6525a01c][ManagementConnection] The reconnection short grace time of 5m has expired, so the connection will be disposed.
[2022-07-28 21:41:10.804] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-28 21:41:18.338] [remoteagent] [error] No ptyHost heartbeat after 12 seconds
[2022-07-28 21:42:41.622] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-28 21:42:49.528] [remoteagent] [error] No ptyHost heartbeat after 12 seconds
[2022-07-28 21:47:54.024] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-28 21:51:55.341] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-28 21:52:01.662] [remoteagent] [error] No ptyHost heartbeat after 12 seconds
[2022-07-28 21:54:00.512] [remoteagent] [info] [127.0.0.1][2550958d][ExtensionHostConnection] <14332> Extension Host Process exited with code: 0, signal: null.
[2022-07-28 21:59:28.668] [remoteagent] [info] [127.0.0.1][99f351a8][ManagementConnection] Another client has connected, will shorten the wait for reconnection 5m before disposing...
[2022-07-28 21:59:28.669] [remoteagent] [info] [127.0.0.1][6ac255e9][ManagementConnection] New connection established.
[2022-07-28 21:59:28.797] [remoteagent] [info] [127.0.0.1][6544d366][ExtensionHostConnection] New connection established.
[2022-07-28 21:59:28.806] [remoteagent] [info] [127.0.0.1][6544d366][ExtensionHostConnection] <15246> Launched Extension Host Process.
[2022-07-28 21:59:30.924] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-28 22:00:07.416] [remoteagent] [info] [127.0.0.1][6544d366][ExtensionHostConnection] <15246> Extension Host Process exited with code: 0, signal: null.
[2022-07-28 22:00:11.867] [remoteagent] [info] [127.0.0.1][08117ea3][ManagementConnection] New connection established.
[2022-07-28 22:00:11.907] [remoteagent] [info] [127.0.0.1][3ab3087e][ExtensionHostConnection] New connection established.
[2022-07-28 22:00:11.911] [remoteagent] [info] [127.0.0.1][3ab3087e][ExtensionHostConnection] <15450> Launched Extension Host Process.
[2022-07-28 22:00:12.366] [remoteagent] [info] [127.0.0.1][6ac255e9][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
[2022-07-28 22:00:12.367] [remoteagent] [error] Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/server/node/server.main.js:145:7933)
    at process.emit (node:events:390:28)
[2022-07-28 22:06:08.994] [remoteagent] [info] [127.0.0.1][08117ea3][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
[2022-07-28 22:06:09.035] [remoteagent] [error] Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/server/node/server.main.js:145:7933)
    at process.emit (node:events:390:28)
[2022-07-28 22:06:10.364] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateIcon (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25409)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-28 22:06:10.364] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-28 22:06:10.367] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-28 22:06:10.390] [remoteagent] [info] [127.0.0.1][6ac255e9][ManagementConnection] Another client has connected, will shorten the wait for reconnection 5m before disposing...
[2022-07-28 22:06:10.390] [remoteagent] [info] [127.0.0.1][08117ea3][ManagementConnection] Another client has connected, will shorten the wait for reconnection 5m before disposing...
[2022-07-28 22:06:10.390] [remoteagent] [info] [127.0.0.1][7dc0bf4a][ManagementConnection] New connection established.
[2022-07-28 22:06:10.496] [remoteagent] [info] [127.0.0.1][d5d23360][ExtensionHostConnection] New connection established.
[2022-07-28 22:06:10.507] [remoteagent] [info] [127.0.0.1][d5d23360][ExtensionHostConnection] <15608> Launched Extension Host Process.
[2022-07-28 22:06:10.957] [remoteagent] [info] [127.0.0.1][3ab3087e][ExtensionHostConnection] <15450> Extension Host Process exited with code: 0, signal: null.
[2022-07-28 22:06:12.836] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-28 22:06:25.157] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-28 22:07:01.120] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-28 22:07:16.599] [remoteagent] [error] No ptyHost heartbeat after 12 seconds
[2022-07-28 22:07:17.618] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateIcon (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25409)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-28 22:07:17.618] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-28 22:07:17.618] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-28 22:07:18.147] [remoteagent] [info] [127.0.0.1][7dc0bf4a][ManagementConnection] The client has reconnected.
[2022-07-28 22:07:18.307] [remoteagent] [info] [127.0.0.1][d5d23360][ExtensionHostConnection] The client has reconnected.
[2022-07-28 22:07:30.089] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-28 22:09:25.683] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-28 22:10:00.193] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-28 22:11:37.224] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-28 22:11:38.230] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-28 22:11:38.235] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateIcon (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25409)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-28 22:11:38.244] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-28 22:11:38.249] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-28 22:11:38.715] [remoteagent] [info] [127.0.0.1][7dc0bf4a][ManagementConnection] The client has reconnected.
[2022-07-28 22:11:38.819] [remoteagent] [info] [127.0.0.1][d5d23360][ExtensionHostConnection] The client has reconnected.
[2022-07-28 22:11:50.162] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-28 22:13:13.194] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-28 22:13:59.075] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-28 22:13:59.099] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateIcon (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25409)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-28 22:13:59.099] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-28 22:13:59.100] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-28 22:13:59.613] [remoteagent] [info] [127.0.0.1][7dc0bf4a][ManagementConnection] The client has reconnected.
[2022-07-28 22:13:59.806] [remoteagent] [info] [127.0.0.1][d5d23360][ExtensionHostConnection] The client has reconnected.
[2022-07-28 22:14:01.697] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-28 22:14:15.643] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-28 22:15:07.806] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-28 22:15:56.810] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-28 22:17:23.707] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-28 22:18:19.456] [remoteagent] [info] [127.0.0.1][7dc0bf4a][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
[2022-07-28 22:18:19.460] [remoteagent] [error] Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/server/node/server.main.js:145:7933)
    at process.emit (node:events:390:28)
[2022-07-28 22:18:19.504] [remoteagent] [info] [127.0.0.1][d5d23360][ExtensionHostConnection] <15608> Extension Host Process exited with code: 0, signal: null.
[2022-07-28 22:18:20.405] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateIcon (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25409)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-28 22:18:20.405] [remoteagent] [error] ["ErrorNoTelemetry: Could not find pty on pty host","    at d._throwIfNoPty (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103)","    at d.updateTitle (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349)","    at Object.call (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293)","    at onPromise (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880)","    at onRawMessage (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268)","    at /home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556","    at h.invoke (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at z.deliver (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at M.fire (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.W (/home/ec2-user/.vscode-server/bin/3b889b090b5ad5793f524b5d1d39fda662b96a2a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-07-28 22:18:48.163] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-07-28 22:22:50.933] [remoteagent] [info] [127.0.0.1][99f351a8][ManagementConnection] The reconnection short grace time of 5m has expired, so the connection will be disposed.
[2022-07-28 22:24:10.452] [remoteagent] [info] [127.0.0.1][6ac255e9][ManagementConnection] The reconnection short grace time of 5m has expired, so the connection will be disposed.
[2022-07-28 22:24:10.453] [remoteagent] [info] [127.0.0.1][08117ea3][ManagementConnection] The reconnection short grace time of 5m has expired, so the connection will be disposed.
[2022-07-28 22:26:06.453] [remoteagent] [info] [127.0.0.1][00fda346][ExtensionHostConnection] <14683> Extension Host Process exited with code: 0, signal: null.
[2022-07-28 22:26:06.454] [remoteagent] [info] Last EH closed, waiting before shutting down
[2022-07-28 22:31:06.454] [remoteagent] [info] Last EH closed, shutting down
